Cargo.toml

```diff
@@ -15,6 +15,7 @@ members = [
   "crypto",                       # Cryptographic primitives and their interfaces.
   "consensus",                    # Consensus related logic.
   "chainstate",                   # Code on chainstate of blocks and transactions.
+  "chainstate_test_framework",    # Test framework for chainstate-related tests.
```

Comment:
I don't think this should be a part of the workspace... it should just be a dependency. Are you sure this is the right way to do this?
For example, chainstate-types isn't here. Because it's just a dependency.